app/models/project_media.rb

```diff
@@ -295,14 +302,42 @@ def replace_by(new_pm, skip_send_report = false)
       Rails.cache.write("check_cached_field:ProjectMedia:#{new_pm.id}:creator_name", 'Import')
 
       # Apply other stuff in background
-      self.class.delay_for(5.seconds).apply_replace_by(self.id, new_pm.id, skip_send_report)
+      options = {
+        author_id: User.current&.id,
+        assignments_ids: assignments_ids,
+        skip_send_report: skip_send_report
+      }
+      self.class.delay_for(5.seconds).apply_replace_by(self.id, new_pm.id, options.to_json)
     end
   end
 
-  def self.apply_replace_by(old_pm_id, new_pm_id, skip_send_report = false)
+  def self.apply_replace_by(old_pm_id, new_pm_id, options_json)
```

Comment:
Question: The old implementation had a default value for the `skip_send_report` argument, and now there is no default value for `options_json` - did you make sure that all replaces that call this method are passing the three arguments?

Reply:
This is the only place that calling `apply_replace_by`   but I'll make sure to run all SideKiq jobs for this one before live deployment